### ISSUE TITLE
MonacoEditor: Upgrade to 0.31.0

### ISSUE THREAD

**Issue by melloware** [primefaces-extensions/primefaces-extensions#651]
@blutorange you can see the latest Monaco is breaking the build:

https://github.com/primefaces-extensions/primefaces-extensions/runs/4506026898?check_suite_focus=true

No rush just if you have time to look at it.

**Comment by blutorange** [comment]
The build output looks like the preprocessing script we use to make localization possible does some wrong replacements. Probably not too hard to fix, I'll take a look at it later

```
[INFO] > function localize('vs/language/html/html.worker', key, message, ...args) {
[INFO] |   return format(message, args);
[INFO] | }
```

**Comment by melloware** [comment]
I just merged 0.31.1
